features/step_definitions/record_of_guidance_contents_steps.rb

```diff
@@ -120,12 +120,7 @@
 end
 
 Then(/^the record of guidance should include their details$/) do
-  output_document = OutputDocument.new(@appointment_summary)
-
-  expected = {
-    attendee_name: output_document.attendee_name,
-    value_of_pension_pots: output_document.value_of_pension_pots
-  }
+  expected = fixture(:populated_csv)
```

Comment:
we should assert only the fields we care about, not the entire csv row, otherwise unrelated changes will fail this assertion